src/docs/implementations/dynatrace.adoc

```diff
@@ -38,12 +37,16 @@ DynatraceConfig dynatraceConfig = new DynatraceConfig() {
 MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
 ----
 
-If you are working with Spring Boot, you can instead put the following properties in your `application-dynatrace.yml` in the `resources` folder.
+`DynatraceConfig` is an interface with a set of default methods.
+Micrometer's Spring Boot support binds properties prefixed with `management.metrics.export.dynatrace` directly to the `DynatraceConfig`.
+This allows configuring the Dynatrace exporter by using the <<bookmark-available-properties, the available properties>>.
+To enable the export to the local OneAgent, use:
 
 [source,yml]
 ----
 management.metrics.export.dynatrace:
     enabled: true
+
```

Comment:
```suggestion
```
Did you intend to remove the line?

Reply:
From my tests, it seems as if this is required.

Reply:
It's only required for the tests bundled with Micrometer. Removed them.